src/files.md

```diff
@@ -64,7 +64,7 @@ syncing in order to use the full sync option without first downloading
 the changes. Similarly, if you are experiencing problems with Anki, you
 might want to (or might be instructed to) disable add-ons temporarily to
 see if one might be causing the problem. You can do both of these things
-by holding down the <kbd>Shift</kbd> key while starting Anki.
+by starting Anki in safe mode, holding down the <kbd>Shift</kbd> key until the on-screen confirmation appears.
```

Comment:
This could've been more like:

```
To do both of these, you can start Anki in safe mode by holding down the <kbd>Shift</kbd> key while opening the app.
```

Reply:
The whole point was to come up with an instruction that _incorporated_ the confirmation. I also wanted the phrasing to match between the 2 sections. It was hard to find a place for both "start Anki" and "while starting Anki" without sounding too redundant. If you really think this needs a "while" phrase, how about this? 

> To do both of these things at the same time, you can open Anki in safe mode by holding down the <kbd>Shift</kbd> key while starting, until the on-screen confirmation appears.

> then open it in safe mode by holding down the <kbd>Shift</kbd> key while starting, until the on-screen confirmation appears.

Reply:
Go ahead. The first paragraph sounds nice.